source/examples/rapids-optuna-hpo/notebook.ipynb

```diff
@@ -28,17 +28,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "## Run this cell to install optuna\n",
-    "# !pip install optuna"
+    "#!pip install optuna optuna-integration"
```

Comment:
This notebook calls the following Python code:

```python
storage = optuna.integration.DaskStorage()
```

The `optuna.integration` module is provided by a separate package, `optuna-integration`, which is not pulled in by default by `pip install optuna`.

Adding this avoids a `ModuleNotFoundError` later on in the notebook.